src/Pact/Native/Trans/TOps.hs

```diff
@@ -0,0 +1,14 @@
+-- |
+-- Module      :  Pact.Native.Trans.TOps
+-- Copyright   :  (C) 2016 Stuart Popejoy
+-- License     :  BSD-style (see the file LICENSE)
+-- Maintainer  :  Stuart Popejoy <stuart@kadena.io>
+--
+-- Operators and math built-ins.
+--
+
+module Pact.Native.Trans.TOps
```

Comment:
Why this module?

Reply:
IIRC this is to maintain the structure we had for the MPFR work.

Reply:
Yes, it's just to abstract the transcendental operators from their implementation, which may change again after this release, depending on whether musl ends up revealing discrepancy due to hardware differences or not.